src/mutation_ui.cpp

```diff
@@ -23,27 +23,37 @@ void draw_exam_window( WINDOW *win, const int border_y )
     mvwputch( win, border_y, width - 1, BORDER_COLOR, LINE_XOXX );
 }
 
-void show_mutations_titlebar( WINDOW *window, std::string &menu_mode )
+const auto shortcut = []( const std::string &s )
 {
-    werase( window );
-
-    std::string caption = _( "MUTATIONS -" );
-    int cap_offset = utf8_width( caption ) + 1;
-    mvwprintz( window, 0,  0, c_blue, "%s", caption.c_str() );
+    return string_format( "<color_yellow>%s</color>", s.c_str() );
+};
 
-    std::string desc;
-    int desc_length = getmaxx( window ) - cap_offset;
+void show_mutations_titlebar( WINDOW *window, std::string &menu_mode, input_context &ctxt )
+{
+    werase( window );
 
+    std::ostringstream desc;
     if( menu_mode == "reassigning" ) {
-        desc = _( "Reassigning.\nSelect a mutation to reassign or press SPACE to cancel." );
-    } else if( menu_mode == "activating" ) {
-        desc = _( "<color_green>Activating</color>  <color_yellow>!</color> to examine, <color_yellow>=</color> to reassign." );
-    } else if( menu_mode == "examining" ) {
-        desc = _( "<color_ltblue>Examining</color>  <color_yellow>!</color> to activate, <color_yellow>=</color> to reassign." );
+        desc << _( "Reassigning." ) << "  " <<
+             _( "Select a mutation to reassign or press <color_yellow>SPACE</color> to cancel. " );
+    }
+    if( menu_mode == "activating" ) {
+        desc << "<color_green>" << _( "Activating" ) << "</color>  " <<
+             string_format( _( "%s to examine mutation, " ),
+                            shortcut( ctxt.get_desc( "TOGGLE_EXAMINE" ).c_str() ).c_str() );
+    }
+    if( menu_mode == "examining" ) {
+        desc << "<color_ltblue>" << _( "Examining" ) << "</color>  " <<
+             string_format( _( "%s to activate mutation, " ),
+                            shortcut( ctxt.get_desc( "TOGGLE_EXAMINE" ).c_str() ).c_str() );
```

Comment:
I see, my fault. How about moving `string_format( ... )` into a function / lambda? Make the `_("%s to examine")` and `"TOOGLE_EXAMINE"` parameters to avoid this confusion.

But there *is* a useless use of `c_str`: `shortcut` takes a `std::string`, it can be called as `shortcut( ctxt.get_desc( "TOGGLE_EXAMINE" ) )`.

Anyhow, a separate function would make this much clearer.